Validation logic applied to edited file

zefciu opened the pull request:
The file that is submitted via `edit` is now subject to validation
logic as any other file. The validation flags were added to the `edit`
command.

Fixes:  #17542

zefciu commented:
Could you please rerun the cla check?

bgrant0607 commented:
@zefciu Thanks for the PR! We're in our "code slush" period for release 1.3:
https://github.com/kubernetes/kubernetes/wiki/Release-1.3#may-23---may-27

We'll review this PR after we cut the 1.3 release branch.

cc @kubernetes/kubectl

zefciu committed:
Validation logic applied to edited file

The file that is submitted via edit is now subject to validation
logic as any other file. The validation flags were added to the edit
command.

Fixes: #17542

jlowdermilk commented:
Change lgtm, needs rebase.

jlowdermilk commented:
@k8s-bot ok to test
